src/ocean/meta/codegen/CTFE.d

```diff
@@ -119,3 +120,52 @@ unittest
     assert (stripQualifiedPrefix("aabb") == "aabb");
     static assert (stripQualifiedPrefix("aabb") == "aabb");
 }
+
+
+/******************************************************************************
+
+    Converts a snake_case name to a PascalCase.
+    This should be used only at CTFE since is not memory efficient.
```

Comment:
Can you describe what an example use case is here?

Reply:
I'm not sure if I can.

Reply:
If your code is following a certain coding standard, and you're also using mixin to generate some of your code, and you want the result of your mixin to also comply with the coding standard, without forcing the client code (which calls your mixin) to pass params to your mixin that follow your coding standard. I think that's the original use case of this case.